src/lib.rs

```diff
@@ -334,7 +431,7 @@ mod tests {
 
     #[test]
     /// Test whether [`bracket`] sequences actions correctly.
```

Comment:
This comment seems redundant now, it's basically the same as the test function name

Reply:
I still like having the comment there tbh